WalletWasabi.Daemon/PersistentConfig.cs

```diff
@@ -126,6 +126,10 @@ public record PersistentConfig : IConfigNg
 	[JsonPropertyName("FeeRateEstimationProvider")]
 	public string  FeeRateEstimationProvider { get; init; } = "BlockstreamInfo";
 
+	[DefaultValue(30)]
+	[JsonPropertyName("DropUnconfirmedTransactionsAfterDays")]
+	public int DropUnconfirmedTransactionsAfterDays { get; init; } = 30;
```

Comment:
Yes, I implemented it in that way first but then I thought it would allow users to set it to 27 seconds or things that would make us spend a lot of time giving support and then changed it. However, it seems ugly to have days. Honestly I don't know, I think I will let it as it is.